articles/language/file-structure.md

```diff
@@ -66,7 +66,7 @@ The `open` directive applies to the entire namespace block.
 For instance,
 
 ```qsharp
-open Microsoft.Quantum.Canon;
+open Microsoft.Quantum.Standard;
```

Comment:
As above, the `Microsoft.Quantum.Canon` namespace is still provided by the `Microsoft.Quantum.Standard` NuGet package.
```suggestion
open Microsoft.Quantum.Canon;
```